src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,34 @@
 import React from 'react';
+import { Comment } from '../../types/types';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = (props) => (
```

Comment:
Always destructure `props`. Fix all cases please
```suggestion
export const CommentInfo: React.FC<Props> = ({ comment }) => (
```